### ISSUE TITLE
Back to single volume

### ISSUE THREAD

**Issue by FelixKlauke** [FelixKlauke/paperspigot-docker#39]
At the moment there are 5 volumes. I would like to merge them back to one to reduce complexity.

**Comment by juresyahya** [comment]
any specific reason why you split it into 5 different volumes in the first place?

**Comment by FelixKlauke** [comment]
TL;DR: Distributed Storage

@juresyahya Yeah that was a specific request of a user. Usually most users will use Volumes as "FIre & Forget". They will be created once and only be used at an active container. This user wanted to edit plugins & configs without mounting a volume in a container. But he still didn't want to deal with worlds. When you have more volumes you can distribute the content. 
You can, for example, use NAS storage for the world but keep the configs in s mounted host folder, so you can edit them more easily. More volumes are more flexible when you want to have control over a specific type of data.

But I'm not sure anymore, that this make it easier.

**Comment by FelixKlauke** [comment]
After further considerations this will be delayed or not even done at all.
